docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -167,6 +167,18 @@ configuration properties as the Hive connector's Glue setup. See
     connector.name=iceberg
     iceberg.catalog.type=glue
 
+.. list-table:: Iceberg Glue catalog configuration properties
+  :widths: 35, 50, 15
+  :header-rows: 1
+
+  * - Property name
+    - Description
+    - Default
+  * - ``iceberg.glue.skip-archive``
+    - Skip archiving an old table version when creating a new version in a
+      commit.
```

Comment:
Maybe link to iceberg docs for this skip archiiving

Reply:
This is the link in Iceberg. 
https://iceberg.apache.org/docs/latest/aws/#skip-archive